config/.credo.exs

```diff
@@ -25,7 +25,8 @@
          ignore_names: [
            ~r/(GlimeshWeb.*)$/,
            ~r/(\.\w+Controller|\.Endpoint|\.Repo|\.Router|\.\w+Socket|\.\w+View|\.\w+Live)$/
-         ]}
+         ]},
+        {Credo.Check.Readability.AliasOrder, false}
```

Comment:
Why remove this?

Reply:
Credo fails on multi alias statments so this has to be toggled off because of it.